core/playwright/fixtures/fields-page.ts

```diff
@@ -31,11 +31,23 @@ export class FieldsPage {
 		await this.page.getByTestId(`select-${format}`).click();
 	}
 
-	async addField(name: string, format: CoreSchemaType) {
+	async addField(
+		name: string,
+		format: CoreSchemaType,
+		/**
+		 * Whether to wait for the little pop up that confirms field creation
+		 * set to false for testing this function
+		 */
+		waitForText = true
+	) {
 		await this.openNewFieldModal();
 		await this.nameBox.fill(name);
 		await this.selectFormat(format);
 		await this.page.getByRole("button", { name: "Create" }).click();
+		if (!waitForText) {
+			return;
+		}
+		await this.page.getByText(`Created field ${name}`, { exact: true }).waitFor();
```

Comment:
for some reason i ran into the problem a lot that the navigation from the field page was too quick, so i added a waitFor